src/bliss/bench_ART.h

```diff
@@ -0,0 +1,49 @@
+#ifndef BLISS_BENCH_ART
+#define BLISS_BENCH_ART
+
+#include <ART.h>
+
+#include <vector>
+
+#include "bliss/bliss_index.h"
+#include "spdlog/spdlog.h"
+
+
+namespace bliss {
+
+template <typename KEY_TYPE, typename VALUE_TYPE>
+class BlissARTIndex : public BlissIndex<KEY_TYPE, VALUE_TYPE> {
+   public:
+   ART::Node* _index;
+    BlissARTIndex() {
+        _index = nullptr;
+    };
+
+    void bulkload(
+        std::vector<std::pair<KEY_TYPE, VALUE_TYPE>> values) override {
+        // expects the pairs to be pre-sorted before performing bulk load
+        for (const auto& pair : values) {
+            put(pair.first, pair.second);
+        }
+    }
+
+    bool get(KEY_TYPE key) override { 
+        uint8_t ARTkey[8];
```

Comment:
8 is a hardcoded number. Why not we move it as a member variable that can be changed? Also, in the current implementation, if key is larger than 8 bytes, this is going to throw an error, and if it is less, we are wasting space.